mautic/mautic
# Fix gated video on landing page preview

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:

Gated video is not display on preview of landing page, because it require mtc.js tracking code. 
My idea is placeholder on preview page

![image](https://user-images.githubusercontent.com/462477/119149712-0c35cd00-ba4e-11eb-8406-2014b859cc35.png)



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create landing page
3. Create gated video
4. Check preview
5. You see nothing
6. After PR you see placholder

![image](https://user-images.githubusercontent.com/462477/119149712-0c35cd00-ba4e-11eb-8406-2014b859cc35.png)
